project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -57,7 +57,7 @@ public static User logInUser(String email, String nickname) {
     }
   }
 
-  public static User getUserByID(String email) {
+  public static User getUserByEmail(String email) {
```

Comment:
I feel like this should be moved to the Database class, or wherever the other "getUserBy.." functions are